core/src/test/java/io/jenkins/pluginhealth/scoring/probes/CodeCoverageProbeTest.java

```diff
@@ -114,7 +114,7 @@ public void shouldBeInErrorWhenRepositoryIsNotInOrganization() {
             Map.of(
                 pluginName, new io.jenkins.pluginhealth.scoring.model.updatecenter.Plugin(
                     pluginName, new VersionNumber("1.0"), scmLink, ZonedDateTime.now(), List.of(), 0,
-                    "42", defaultBranch
+                    "42", defaultBranch, List.of()
```

Comment:
Looks like you had to modify a lot of similar data? Maybe it's worth extracting somewhere so you would only have 1 place to update?